app/extension/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (C) 2016 Red Hat, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.syndesis</groupId>
+    <artifactId>syndesis-parent</artifactId>
+    <version>1.2-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <groupId>io.syndesis.extension</groupId>
```

Comment:
I wonder whether we should introduce a new group. I know we already have a kind of mixture, but I think having one single group `io.syndesis` has some benefits, as now the artifactId need to be unique (a counter example is "model" which exists for 'io.syndesis' and 'io.syndesis.runtime'.

Alternatively we could decide to use a specific group for every top-level module.

@syndesisio/backend wdyt ?

Reply:
No preference about that. Probably `io.syndesis.extension` may be redundant since all artifacts in that module start with the prefix `extension-`.. Changing it should be a simple find/replace..

Reply:
Ok, then let's go with a top-level 'io.syndesis'. This forces us to make the artifactId unique by adding the top-level module as prefix. 

The next step is then get the naming right with #1078 , where we still could reconsider the group naming.

@nicolaferraro are you fine with replacing the group ?

Reply:
Ok, I will replace the groupId with `io.syndesis` soon.